.github/workflows/test-daap.yml

```
@@ -0,0 +1,39 @@
+---
+name: Test application code
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - containers/**
+  push:
+    paths:
+      - containers/**
+
+jobs:
+  run-tests:
+    name: Test python
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        container:
+          - daap-get-glue-metadata
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.11
+          cache: pip
+          cache-dependency-path: containers/${{ matrix.container }}/tests/requirements.txt
+
+      - name: Install test dependencies
+        run: python -m pip install -r requirements.txt
+        working-directory: containers/${{ matrix.container }}/tests/
+
+      - name: Run tests
+        run: pytest
```

could we increase verbosity of output?
```suggestion
        run: pytest -vv